src/dev/src/app/datagrid/disabled-rows/disabled-rows.ts

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright (c) 2016-2019 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+import { Component } from '@angular/core';
+
+import { Inventory } from '../inventory/inventory';
+import { User } from '../inventory/user';
+
+@Component({
+  selector: 'clr-datagrid-disabled-rows-demo',
+  providers: [Inventory],
+  templateUrl: './disabled-rows.html',
+  styleUrls: ['../datagrid.demo.scss'],
+})
+export class DatagridDisabledRowsDemo {
+  users: User[];
+  usersMulti: User[] = [];
+  selected: User;
+  selectedRows: User[] = [];
+
+  constructor(inventory: Inventory) {
+    inventory.size = 10;
+    inventory.reset();
+    this.users = inventory.all;
+    this.usersMulti = inventory.all;
+  }
+
+  unlockRow() {
+    this.users = this.users.map(row => {
+      row.locked = undefined;
```

Comment:
It seems strange to set to 'undefined' instead of to `false`. You could also `delete row.locked;` if you want to remove it.